src/main/kotlin/io/github/mojira/arisa/ArisaMain.kt

```diff
@@ -59,6 +60,14 @@ fun main() {
 
             lastRunFile.writeText("${curRunTime.toEpochMilli()}$failed")
             lastRunTime = curRunTime
+        } else if (lastRelog.plus(1, ChronoUnit.MINUTES).isAfter(Instant.now())) {
```

Comment:
Maybe it would be better to do that only if the last successful run was more than a minute ago?
There are sometimes hiccups where a run fails, but the next run 10 seconds later succeeds again, without a restart or anything.